packages/sku/README.md

```diff
@@ -0,0 +1 @@
+../../README.md
```

Comment:
This is a symlink to the root readme. However, this results in the readme no being present in the `sku` package. I'm not too fussed about this, but if we want to keep the readme in the built package, we could reverse the symlink, just copy paste, or maybe have different readmes bewteen the package and the monorepo root.

Reply:
Given that sku [sub-package] is the main character, I think we should reverse the symlink.

Reply:
The logo link is relative, so it'll only work in one place, not both 😢. Could just hardcode it to the raw URL, but that's not ideal.

Reply:
Actually if it's just relative it just needs to change to be from the sku package instead. 🤦

Reply:
Github doesn't know how to resolve it I guess
![image](https://user-images.githubusercontent.com/5663042/227424747-b2559197-4351-449e-bc0e-87a9551265af.png)

Reply:
I'd be ok with having the full path to the file i.e. `https://github.com/seek-oss/sku/raw/master/docs/logo/logo.png?raw=true`